### DISCUSSION TITLE
tag/v0.5.0

### DISCUSSION THREAD

**Discussion by xjmzx** [bitcoinresearchkit/brk#8] [General]
Has there been some change not reflected in the install docs?
I don't see a parser directory. Running this from scratch on slightly different hardware.

**Comment by nym21** [comment]
On the the latest commit on the main branch yes, the README isn't up to date yet, it's a work in progress

If you want the v0.5.0 you should go to the [tags/releases](https://github.com/kibo-money/kibo/tags) and download it [from there](https://github.com/kibo-money/kibo/releases/tag/v0.5.0), either via a [zip file](https://github.com/kibo-money/kibo/archive/refs/tags/v0.5.0.zip) or by going to [the state of the repo at that time](https://github.com/kibo-money/kibo/tree/eea56d394bf92c62c81da8b78b8c47ea730683f5)

**Comment by xjmzx** [comment]
Understood and thanks for explaining. I'm obviously aware this is all a work in progress. Running things from scratch, this time with all the data on the same NVMe disk to see how that performs.

Sent over a small donation, good luck with the development.

**Reply by nym21** [reply]
No worries and you can still use the latest code if you want, I'll try to update to README asap but in the meantime `cargo run -r -- -help` will be your friend and of course you ask questions here too

Few nice things about the latest code is that it's only one package now instead of two separate and the defragmentation is automatic now too

Thank you !

**Reply by nym21** [reply]
@xjmzx readme up to date 🤙
